docker/Dockerfile

```diff
@@ -76,9 +76,9 @@ RUN export DEBIAN_FRONTEND=noninteractive && apt update && \
 
 # grab the builder stages artifacts
 COPY --from=builder /usr/local /usr/local
-COPY --from=builder /usr/local/lib/python3.12/dist-packages/valhalla/* /usr/local/lib/python3.12/dist-packages/valhalla/
+COPY --from=builder /usr/local/lib/python3.12/dist-packages/valhalla /usr/local/lib/python3.12/dist-packages/valhalla/
```

Comment:
note the removal of the asterisk. that's all it was.

Reply:
it might be. for files [it is](https://docs.docker.com/reference/dockerfile/#destination-1):

> Trailing slashes are significant. For example, COPY test.txt /abs creates a file at /abs, whereas COPY test.txt /abs/ creates /abs/test.txt.

not sure if for directories. anyways, this way it works and I wouldn't remember the answer next night's sleep haha